src/control/cmd/daos_server/main.go

```diff
@@ -123,6 +123,8 @@ func parseOpts(args []string, opts *mainOpts, log *logging.LeveledLogger) error
 					return errors.Wrap(err, "failed to set CLI config overrides")
 				}
 			}
+		} else if opts.ConfigPath != "" {
+			log.Infof("DAOS Server config has been supplied but this command will not use it")
 		}
```

Comment:
shouldn't we raise an error rather than just logging an info message?

Reply:
that's an open question, I erred on the side of permissiveness in order to prevent inconvenience, it doesn't cause a problem providing config but the values are ignored so I decided an informational message was sufficient, @mjmac @johannlombardi  thoughts? happy to change it to an error if that's thought to be more appropriate

Reply:
I think it should probably be an error, as the admin may not notice the info message mixed in with the rest of the output. I can see both sides of this, but I think I agree with Michael here because the downside of allowing it to just proceed is that the behavior may be confusing.

Reply:
The repercussion may be that people ask why is the command option (-o) listed in the usage output if it's an unsupported option.